pkg/plugins/filter/pd_role.go

```diff
@@ -0,0 +1,43 @@
+package filter
+
+import (
+	"encoding/json"
+
+	"sigs.k8s.io/gateway-api-inference-extension/pkg/epp/plugins"
+)
+
+const (
+	// RoleLabel name
+	RoleLabel = "llm-d.ai/role"
+	// RolePrefill set for designated prefill workers
+	RolePrefill = "prefill"
+	// RoleDecode set for designated decode workers
+	RoleDecode = "decode"
+	// RoleBoth set for workers that can act as both prefill and decode
+	RoleBoth = "both"
+
+	// DecodeType is the type of the DecodeFilter
```

Comment:
- see note below regarding adding Role to the filter name
- split DecodeFilter to "Decode filter" (same for prefill) in the comment